test/extended/testdata/gssapi/fedora/base/gssapi-tests.sh

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/bash
+
+set -e
+set -x
+
+echo "I ran this on `uname -a`"
+
+CLIENT_HAS_GSSAPI=true
+SERVER_HAS_BASIC=true
+users=(user1 user2 user3 user4 user5)
+
+# Client has no GSSAPI and server is GSSAPI only
+# Everything fails
+
+if [[ !CLIENT_HAS_GSSAPI && !SERVER_HAS_BASIC ]]
+then
+    for u in ${users[@]}
+    do
+        u+=$realm
```

Comment:
we want to make sure we can kinit with and without the realm, and `oc login -u <user>` with and without the realm